DetoxSync/DetoxSync/Utils/_DTXTimerTrampoline.m

```diff
@@ -192,12 +169,12 @@ + (NSDateFormatter*)_descriptionDateFormatter
 }
 
 - (DTXBusyResource *)jsonDescription {
-  return @{
-    @"fire_date": _fireDate ? [_DTXTimerTrampoline._descriptionDateFormatter stringFromDate:_fireDate] : @"none",
-    @"time_until_fire": @(_timeUntilFire),
-    @"is_recurring": @(_repeats),
-    @"repeat_interval": @(_ti)
-  };
+	return @{
+		@"fire_date": _fireDate ? [_DTXTimerTrampoline._descriptionDateFormatter stringFromDate:_fireDate] : @"none",
+		@"time_until_fire": @(_timeUntilFire),
+		@"is_recurring": @(_repeats),
+		@"repeat_interval": @(_ti)
+	};
```

Comment:
The rest of the file is indented using tabs. Fixed for consistency (and editor happiness).

Reply:
Legit :+1:
To be honest, I hate tabs, but I prefer consistency over inconsistency.